app/src/main/res/layout/setting_fragment.xml

```diff
@@ -240,7 +240,7 @@
                 android:textSize="20sp" />
 
             <ImageButton
-                android:id="@+id/setting_plus_weight"
+                android:id="@+id/plus_weight"
```

Comment:
There os an approach to avoid view name duplication at all. So, if you already had screen name in variable, it's more that fine to leave it as it was.

Reply:
Just info, no need to revert.